test/io/CsvWriterTest.ooc

```diff
@@ -30,7 +33,21 @@ CsvWriterTest: class extends Fixture {
 			}
 			reader free()
 			outputFilename free()
+			// This needs to be moved if any more tests are implemented
+			this _removeOutputDirectory()
```

Comment:
Instead of having to remove each file after each test, we wipe the entire directory. You just have to make sure that ```this removeOutputDirectory``` is the last instruction in the last test. There won't be that many tests for this particular class anyway.